activity.py

```diff
@@ -8,52 +8,38 @@
 import sugargame.canvas
 from sugar3.activity import activity
 from sugar3.graphics.toolbarbox import ToolbarBox
-from sugar3.activity.widgets import ActivityToolbarButton
+from sugar3.activity.widgets import ActivityToolbarButton, StopButton
 from sugar3.graphics.radiotoolbutton import RadioToolButton
 from sugar3.graphics.toolbutton import ToolButton
-from sugar3.activity.widgets import StopButton
 from gettext import gettext as _
 import main
 import pygame
 
 
 class Activity(activity.Activity):
-
     def __init__(self, handle):
         activity.Activity.__init__(self, handle)
         self.max_participants = 1
         self.sound = True
         self.game = main.Flappy()
         self.build_toolbar()
-        self.game.canvas = self._pygamecanvas = \
-            sugargame.canvas.PygameCanvas(self,
-                                          main=self.game.run,
-                                          modules=[pygame.display, pygame.font,
-                                                   pygame.mixer])
-        self.set_canvas(self._pygamecanvas)
-        self._pygamecanvas.grab_focus()
+        self.game.canvas = sugargame.canvas.PygameCanvas(self, main=self.game.run, modules=[pygame.display, pygame.font, pygame.mixer])
+        self.set_canvas(self.game.canvas)
+        self.game.canvas.grab_focus()
 
     def build_toolbar(self):
-
         toolbar_box = ToolbarBox()
+
         activity_button = ActivityToolbarButton(self)
         toolbar_box.toolbar.insert(activity_button, -1)
-        activity_button.show()
 
-        separator = Gtk.SeparatorToolItem()
-        separator.props.draw = False
-        separator.set_expand(False)
+        separator = Gtk.SeparatorToolItem(draw=False, expand=False)
         toolbar_box.toolbar.insert(separator, -1)
-        separator.show()
 
         self._levels_buttons = []
 
         def add_level_button(icon_name, tooltip, numeric_level):
-            if self._levels_buttons:
-                button = RadioToolButton(icon_name=icon_name,
-                                         group=self._levels_buttons[0])
-            else:
-                button = RadioToolButton(icon_name=icon_name)
+            button = RadioToolButton(icon_name=icon_name, group=self._levels_buttons[0] if self._levels_buttons else None)
```

Comment:
Did you confirm that the test case was unnecessary? Also this isn't pep8 compliant.

Reply:
my apologies for missing that.